stocktrader/loyalty_deploy.yaml

```diff
@@ -33,23 +33,41 @@ spec:
     spec:
       containers:
       - name: loyalty-level
-        image: kappnav/giddyup:latest
```

Comment:
Only specify docker.io in the image name if you really want to restrict it to that location.

Reply:
done! -thanks!